seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/master/JobMaster.java

```diff
@@ -222,6 +222,8 @@ public void init(long initializationTimestamp, boolean restart) throws Exception
                         runningJobStateTimestampsIMap,
                         engineConfig.getQueueType(),
                         engineConfig);
+        // revert to app class loader, it may be changed by PlanUtils.fromLogicalDAG
+        Thread.currentThread().setContextClassLoader(appClassLoader);
```

Comment:
also fix sometimes classloader not revert to app classloader, because the thread class loader may be changed by connector code.